### ISSUE TITLE
The final “result” named video has same face as “data_dst” face

### ISSUE THREAD

**Issue by pardeeprooprai** [iperov/DeepFaceLab#691]
I used Quick96 method on 30.03.2020 version to train upto 170k iterations but after hitting “merged to mp4” I just get same video without any face swapped at all. 

Please if anybody has solution to this issue, help me. Thanks.

**Comment by glueydoob1** [comment]
i have the exact same problem!

**Comment by pardeeprooprai** [comment]
> i have the exact same problem!

I can feel your pain of waiting so long training the software and then getting same video as the result. 

Couldn’t find any method to fix it yet.

**Comment by rocketsvm** [comment]
> train upto 170k iterations but after hitting “merged to mp4”
Just to clarify: you _did_ ran "merge Quick96", right?

**Comment by pardeeprooprai** [comment]
> > train upto 170k iterations but after hitting “merged to mp4”
> 
> Just to clarify: you _did_ ran "merge Quick96", right?

Yes I ran "merge Quick96" and that gave me merged and marged_mask subfolders inside data_dst folder.

**Comment by rocketsvm** [comment]
I just checked the whole process with Quick96. It's flawless. Frames in "merged" should contain frames with replaced face, "merged_mask" should not be completely black.
I think you missed some steps. Did you use interactive merger?

**Comment by pardeeprooprai** [comment]
@rocketsvm I appreciate you going through Quick96 process to help me finding solution. I am sorry but I don't know what interactive merger you asking about.

I simply clicked "Merge Quick96" and after that it asked me which Trained Model I wanted to merge, after that another pop-up window appears for color correction of face and after that merging process starts.

**Comment by rocketsvm** [comment]
@pardeeprooprai That pop-up window is an interactive merger. You probably disabled an overlay there. Read the great guide here [https://mrdeepfakes.com/forums/thread-guide-deepfacelab-2-0-explained-and-tutorials-recommended](url), section called "7. Merging".

**Comment by pardeeprooprai** [comment]
@rocketsvm Thanks for your suggestion but no luck. The result video is same, I even tried getting final video without using interactive merger but still the face in “result” video was the same face from original destination video. 

So basically I am getting the same video back as an output, which I provided as an input. Confusing (and funny) 🤨

**Comment by ShionWakanae** [comment]
> @rocketsvm Thanks for your suggestion but no luck. The result video is same, I even tried getting final video without using interactive merger but still the face in “result” video was the same face from original destination video.
> 
> So basically I am getting the same video back as an output, which I provided as an input. Confusing (and funny) 🤨

I guess, mayby the "learned mask" is not working. that's the default mask used by interactive merger. 

Somehow when that happened, you should switch to other mask type by press "x" on the preview windows  (or called main windows?), and watch that frame picture change, make sure the frame's dst face is replaced, also you may changed another params to make face look more natural.

And don't forget to press Shift+? to override the params to rest frames. then press Shift+> to process them all.

Any way, in the interactive merger, what you see is what you get.

I hope you could understand me, 毕竟我平时不说英文的(from non-English speaking...), and I wish this could be help.

**Comment by JDTCC** [comment]
Does this issue still persist? If not, would you mind closing it?
